pkgs/os-specific/linux/game-devices-udev-rules/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+}:
+stdenv.mkDerivation rec {
+  pname = "game-devices-udev-rules";
+  version = "0.21";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "fabiscafe";
+    repo = "game-devices-udev";
+    rev = version;
+    hash = "sha256-Yy91yDF5BSDTTlr/Pj8e0UklPooEdzvRW8mkhdHtHVo=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    install -D 71-8bitdo-controllers.rules $out/lib/udev/rules.d/71-8bitdo-controllers.rules
+    install -D 71-alpha_imaging_technology_co-vr.rules $out/lib/udev/rules.d/71-alpha_imaging_technology_co-vr.rules
+    install -D 71-astro_gaming-controllers.rules $out/lib/udev/rules.d/71-astro_gaming-controllers.rules
+    install -D 71-betop-controllers.rules $out/lib/udev/rules.d/71-betop-controllers.rules
+    install -D 71-hori-controllers.rules $out/lib/udev/rules.d/71-hori-controllers.rules
+    install -D 71-htc-vr.rules $out/lib/udev/rules.d/71-htc-vr.rules
+    install -D 71-logitech-controllers.rules $out/lib/udev/rules.d/71-logitech-controllers.rules
+    install -D 71-mad_catz-controllers.rules $out/lib/udev/rules.d/71-mad_catz-controllers.rules
+    install -D 71-microsoft-controllers.rules $out/lib/udev/rules.d/71-microsoft-controllers.rules
+    install -D 71-nacon-controllers.rules $out/lib/udev/rules.d/71-nacon-controllers.rules
+    install -D 71-nintendo-controllers.rules $out/lib/udev/rules.d/71-nintendo-controllers.rules
+    install -D 71-nvidia-controllers.rules $out/lib/udev/rules.d/71-nvidia-controllers.rules
+    install -D 71-pdp-controllers.rules $out/lib/udev/rules.d/71-pdp-controllers.rules
+    install -D 71-personal_communication_systems_inc-controllers.rules $out/lib/udev/rules.d/71-personal_communication_systems_inc-controllers.rules
+    install -D 71-powera-controllers.rules $out/lib/udev/rules.d/71-powera-controllers.rules
+    install -D 71-razer-controllers.rules $out/lib/udev/rules.d/71-razer-controllers.rules
+    install -D 71-sony-controllers.rules $out/lib/udev/rules.d/71-sony-controllers.rules
+    install -D 71-sony-vr.rules $out/lib/udev/rules.d/71-sony-vr.rules
+    install -D 71-valve-controllers.rules $out/lib/udev/rules.d/71-valve-controllers.rules
+    install -D 71-valve-vr.rules $out/lib/udev/rules.d/71-valve-vr.rules
+    install -D 71-zeroplus_technology_corporation-controllers.rules $out/lib/udev/rules.d/71-zeroplus_technology_corporation-controllers.rules
```

Comment:
```suggestion
  install -Dm444 -t "$out/lib/udev/rules.d" *.rules
```

Reply:
Yep, that's certainly **way** better than mine! The reason I separated them was for anyone who didn't want a particular rule they could overlay it out, but I think they could just create their own rule that precedes these ones instead.